packages/tools/accessibility/README.md

```diff
@@ -0,0 +1,88 @@
+
+Example:
+https://playground.babylonjs.com/?snapshot=refs/pull/12074/merge#C9GZTF#1
+
+## How to use Accessibility Package to Support Screen Reader and Keyboard Navigation
+
+### IAccessibilityTag
+
+Blind or low vision users listen to the content and use keyboard to interact, so we need to add description to BabylonJS content that will be read by screen reader. You can add description to your Nodes or Controls (that you think should be accessible) by using **IAccessibilityTag**.
+
+```javascript
+let egg = BABYLON.MeshBuilder.CreateSphere("Egg", {diameterX: 0.62, diameterY: 0.8, diameterZ: 0.6}, scene);
+egg.accessibilityTag = {
+    description: "An easter egg"
+}
+```
+
+Not all content in the scene should be accessible. For example, the decorative trees or background image on a UI panel. Only add IAccessibilityTag to the contents that's important to the user experience.
+
+By default, all Controls (GUI) are considered "important" for accessibility, and will be rendered in accessibility tree with its own information (like text to show on a button), even you hasn't assigned the IAccessibility tag. But if you define the IAccessibilityTag, it can override default metadata (like assigning the IAccessibilityTag.description will override the text to show on a button). By default, Node type object are not considered "important", unless you assign a IAccessibilityTag to it.
+
+### RenderAccessibilityTree()
+
+The accessibility package basically generates HTML elements for your scene, based on the metadata (IAccessibilityTag) you added to your scene content. The screen reader then can read the HTML elements. To generate it for your scene:
+
+```javascript
+ACCESSIBILITY.AccessibilityRenderer.RenderAccessibilityTree(scene);
+```
+
+This will generate a `<div id="accessibility-tree">` HTML element right after the babylonJS scene's canvas element. Inside this div element, the renderer generate an HTML twin for each of your accessible contents in the scene. These HTML twin elements are internally connected with BabylonJS objects, so the screen reader user can also interact with the BabylonJS objects through their HTML twin.
+
+### Interaction
+
+Some of your contents in the scene might be interactable (e.g. clickable). For Node type objects, if you use BabylonJS's [ActionManager](https://doc.babylonjs.com/divingDeeper/events/actions) to define the interaction, the Accessibility package can automatically detects it and apply on the generated HTML twin elements, so that the user can use keyboard to trigger events like click or right click on the HTML twin elements, thus trigger the correspond action on the BabylonJS objects. Only ACTION_OnPickTrigger, ACTION_OnLeftPickTrigger, ACTION_OnRightPickTrigger are supported. For Control type objects, if you defined observer for onPointerClickObservable, the Accessibility package can also automatically detects it and apply on the generated HTML twin element.

```

Comment:
"BabylonJS's" should be "Babyon.js's", anywhere you use "BabylonJS" we should change to "Babylon.js" for branding consistency.
and
"...Accessibility package can automatically detects it and apply..." should be 
"...Accessibility package can automatically **detect** it and apply..."
and 
"...onPointerClickObservable, the Accessibility package can also automatically detects it and apply..." should be 
"...onPointerClickObservable, the Accessibility package can also automatically **detect** it and apply..."